src/config.js

```diff
@@ -66,17 +72,98 @@ export default class Config {
 	 * @param {*} [value=null] The configuration value. Used if a name is passed to nameOrConfigurations.
 	 */
 	set( name, value ) {
```

Comment:
> That's all we need so far. Sounds like we need only this Config( userConfig ) and Config#define( [path?], defaults ).
@Reinmar I decided to keep `set` method because implementation looks very easy, but If you are sure that we don't need it I will remove it.